--- ./a/ppdc/Makefile	2020-04-27 23:04:29.000000000 +0500
+++ b/ppdc/Makefile	2026-01-08 22:12:56.349080587 +0500
@@ -182,12 +182,7 @@
 
 genstrings:		genstrings.o libcupsppdc.a ../cups/$(LIBCUPSSTATIC) \
 			sample.drv ../data/media.defs
-	echo Linking $@...
-	$(LD_CXX) $(ARCHFLAGS) $(ALL_LDFLAGS) -o genstrings genstrings.o \
-		libcupsppdc.a $(LINKCUPSSTATIC)
-	$(CODE_SIGN) -s "$(CODE_SIGN_IDENTITY)" $@
-	echo Generating localization strings...
-	./genstrings >sample.c
+	echo Not Linking $@...
 
 
 #
